src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -240,11 +240,6 @@ public function setValue(&$objectOrArray, $propertyPath, $value)
             }
         } catch (\TypeError $e) {
             self::throwInvalidArgumentException($e->getMessage(), $e->getTrace(), 0);
-        } finally {
-            if (PHP_VERSION_ID < 70000 && false !== self::$previousErrorHandler) {
-                restore_error_handler();
-                self::$previousErrorHandler = false;
```

Comment:
no indeed, removed `self::$errorHandler`and `handleError()` which were unused also